tests/api/test_list_messages.py

```diff
@@ -517,7 +516,7 @@ def instance_message_fixture() -> MessageDb:
                     )
                 ),
                 persistence=VolumePersistence("host"),
-                size_mib=PersistentVolumeSizeMib(1024),
```

Comment:
I'm confused why this has been removed?

Reply:
I don't know, was removed from `aleph_message` directly 😕